system/Commands/Server/rewrite.php

```diff
@@ -24,7 +24,9 @@
     return;
 }
 
-$uri = urldecode(parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH));
+$uri = urldecode(
+    parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH) ?? ''
```

Comment:
Is this better?
1. we should not pass a relative URL to `parse_url()`.
2. `parse_url()` might return null.

```suggestion
    parse_url('http://ci4.dev' . $_SERVER['REQUEST_URI'], PHP_URL_PATH) ?? ''
```